src/mapml.css

```diff
@@ -385,6 +385,11 @@
   display: inline;
 }
 
+:host(.mapml-fullscreen-on) .mapml-debug-grid {
```

Comment:
This may as well be `:host(:fullscreen) .mapml-debug-grid`, but changing for consistency.

Reply:
(and https://caniuse.com/mdn-css_selectors_fullscreen)